config/hypr/hyprland.conf

```diff
@@ -1,5 +1,12 @@
 # Learn how to configure Hyprland: https://wiki.hyprland.org/Configuring/
 
+# Change your own setup in these files (and overwrite any settings from defaults!)
```

Comment:
I was thinking, why not create an empty `userconfig.conf` and let the user put their custom configuration in it? This file will never be modified during updates, which should prevent configuration overriding.

It's just an idea.